### PR TITLE
Update MediaUrlList - Twitch.as

### PR DESCRIPTION
**Pull request by jeongminkim-islab**
Fix issue for play twitch livestream.

### CODE REVIEW

**Comment by jeongminkim-islab** [comment]
Patches for this issue.
https://github.com/23rd/TwitchPotPlayer/issues/13

File: Media/UrlList/MediaUrlList - Twitch.as [hunk 1]
```diff
@@ -74,8 +85,10 @@ array<dictionary> GetChunkOfUsersOnline(string allFollowersIds, string header) {
 			//Set every online channel in list of urls.
 			for (int k = 0, lenNames = streams.size(); k < lenNames; k++) {
 				string isPlaylist = streams[k]["type"].asString();
-				string viewers = streams[k]["viewer_count"].asString();
-				string login = streams[k]["user_name"].asString();
+				string viewers = streams[k]["viewer_count"].asString();
```
Comment (23rd) [inline Media/UrlList/MediaUrlList - Twitch.as]: @jeongminkim-islab Could you remove the blank spaces at the end of the line?

Comment (jeongminkim-islab) [reply Media/UrlList/MediaUrlList - Twitch.as]: Okey. I'll fix that.

Comment (23rd) [reply Media/UrlList/MediaUrlList - Twitch.as]: Okay, nevermind, I'll do it in the next commit.

**Comment by 23rd** [comment]
Thanks for your issue.

**Pull request merged**